### ISSUE TITLE
GELI encryption versus boot from GPT (fresh encrypted installations can not boot): FreeBSD bug 213491 affecting many people should be amongst Ongoing TrueOS issues

### ISSUE THREAD

**Issue by grahamperrin** [trueos/trueos-docs#25]
## Background

FreeBSD regression [213491 – gpt{,zfs}boot after r305353 crash in GELIBoot](https://bugs.freebsd.org/bugzilla/show_bug.cgi?id=213491)

> Affects Many People 

At https://discourse.trueos.org/t/-/118/2 under **GELI encryption on install caused "BTX halted" on first boot** – before the list of known issues shifted from the downloads page, to the guide: 

> … please consider adding this to the list of known issues …
> 
> In plain English, maybe:
> 
> - fresh encrypted installations can not boot

More specifically, _to the best of my knowledge_ (please correct me if I'm wrong) the bug: 

- is limited to fresh encrypted installations where the FreeBSD loader is used with GPT (GUID Partition Table)
- does **not** affect fresh encrypted installations where TrueOS-customised GRUB is used with MBR. 

More recent discussion, lack of clarity: 

- https://discourse.trueos.org/t/-/205/4 under **Full disk encryption**

– I'll add a link from there, to this issue and then maybe someone with certain knowledge of the situation can post something in that topic. 

## Attention to the guide

https://www.trueos.org/handbook/introduction.html#ongoing-trueos-issues should gain a bullet point with _something like_ a plain English explanation of: 

- the regression in FreeBSD and its impact on some types of encrypted installation of TrueOS.

Further: given the likelihood of newcomers to TrueOS wasting time with full installations that will not boot, I reckon that there's a good case to _exceptionally_ offer forewarnings in relevant parts of the guide, **especially** at or around https://www.trueos.org/handbook/install.html#install11 where we currently have a screenshot of the invitation to encrypt when using GPT.

## Side notes

Does the guide have a screenshot of GELI encryption with TrueOS-cusomised GRUB? (Sorry … losing track of things here.)

We also need a separate issue, an enhancement request, for TrueOS Disk Wizard to **cease implying that GELI requires GPT**. 

Thanks :-)

**Comment by Mrt134** [comment]
Added, thanks!
